### ISSUE TITLE
lodash/prefer-noop

### ISSUE THREAD

**Issue by Lalem001** [wix/eslint-plugin-lodash#66]
When creating a `noop` in an ES6 class, this rule is triggered:

```javascript
class foo {
  bar() {} // Prefer _.noop over an empty function
}
```

I do not know of a way to assign `_.noop` while maintaining this class syntax.

Thoughts?

**Comment by ganimomer** [comment]
You could _technically_ add it to `foo.prototype`, but I see your point. I wouldn't like to do it either.
Fixing the rule to not warn in this case.

**Comment by Lalem001** [comment]
👍 Thanks!
